client/packages/system/src/Item/context/useUnitVariant.ts

```diff
@@ -98,14 +86,14 @@ export const useUnitVariant = (
   variantsControl?: VariantControl;
   unitVariantsExist: boolean;
 } => {
-  const [item, userSelectedVariantId, setUserSelectedVariant] = useUnitStore(
-    state => [
-      state.items[itemId],
-      state.userSelectedVariants[itemId],
-      state.setUserSelectedVariant,
-    ],
-    isEqual
+  const authContext = useAuthContext();
+  const [userSelectedVariants, setUserSelectedVariant] = useLocalStorage(
+    `/user/${authContext.user?.id ?? '?'}/store/${
+      authContext.storeId
+    }/selectedVariants`
   );
+  const userSelectedVariantId = userSelectedVariants?.[itemId];
+  const [item] = useUnitStore(state => [state.items[itemId]], isEqual);
```

Comment:
I guess I could simplify this?

Reply:
Just removing the no-longer-required array would be fine I think:
```tyepscript
const item = useUnitStore(state => state.items[itemId], isEqual);
```